src/app/components/feed/FeedContent.js

```diff
@@ -68,12 +55,12 @@ const FeedContent = ({
 
 FeedContent.defaultProps = {
   listId: null,
-  dataPoints: [],
+  listType: null,
 };
 
 FeedContent.propTypes = {
   listId: PropTypes.number,
-  dataPoints: PropTypes.arrayOf(PropTypes.number.isRequired),
+  listType: PropTypes.string,
```

Comment:
What are the possible values for `listType`?
Perhaps something like `listType: PropTypes.oneOf(['media', 'article'])` would be a better propType definition

Reply:
Suggestion: renaming this component `FeedContent` => `FeedSelectList` to better reflect what it does